test/PreAMMBatcher-e2e.ts

```diff
@@ -257,4 +257,87 @@ describe("PreAMMBatcher-e2e", () => {
     expect(testCase.solution.sellOrdersToken1.length).to.be.equal(2);
     await runScenarioOnchain(testCase);
   });
+  it("isSorted", async () => {
```

Comment:
nit: I find this test a little too big. Also, to me these are unit tests and not e2e tests.
```suggestion
  describe("PreAMMBatcher.isSorted", () => {
    it("returns true for sorted orders", async () => {
      // ...
    })
    it("returns true for empty list", async () => {
      // ...
    })
    // ...
  })
```

Reply:
This change is coming in the clean up PR #15

Reply:
That is, I would like to merge this first and then include your suggestion here in the cleanup branch that does this for all existing unit tests

Reply:
I don't see the issue with just doing this test this PR and doing the other ones in #15 so that it #15 is smaller. But since you already did it there, thats OK too I guess.

Reply:
Glad its ok, gonna stick with it.